telegram/ext/conversationhandler.py

```diff
@@ -423,6 +444,45 @@ def _get_key(self, update: Update) -> Tuple[int, ...]:
 
         return tuple(key)
 
+    def _resolve_promise(self, state: Tuple) -> object:
```

Comment:
```suggestion
    def _resolve_promise(self, state: Tuple[object, Promise]) -> object:
```

Reply:
I did same at first but mypy started raising:
```python
telegram/ext/conversationhandler.py:520: error: Argument 1 to "_resolve_promise" of "ConversationHandler" has incompatible type "Tuple[Any, ...]"; expected "Tuple[object, Promise]"  [arg-type]
```
Even after having `isinstance(state, tuple) and isinstance(state[1], Promise)` check in `handle_update` method, since `_resolve_promise()` was private method i just added `Tuple` after that.

Reply:
ah, because `self.conversations` is typed to be `Dict[Tuple[int, …], Optional[object]]` . Well, then let's just leave it this way. It's not important.